### PR TITLE
Fix compilation warnings  

### PR DESCRIPTION
**Pull request by swapnili**

### CODE REVIEW

File: lib/muser_priv.h [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 #include "muser.h"
 
+#define UNUSED __attribute__((unused))
```
Comment (tmakatos) [inline lib/muser_priv.h:38]: Why not just drop this define?

File: samples/client.c [hunk 2]
```diff
@@ -548,7 +548,8 @@ get_dirty_bitmaps(int sock, struct vfio_user_dma_region *dma_regions,
 {
     struct vfio_iommu_type1_dirty_bitmap dirty_bitmap = {0};
     struct vfio_iommu_type1_dirty_bitmap_get bitmaps[2];
-    int ret, i;
+    int ret;
+	size_t i;
```
Comment (tmakatos) [inline samples/client.c:552]: identation

**Review by tmakatos — COMMENTED** [review]
couple of minor comments, good to merge anyway

File: lib/muser_priv.h (hunk 1 above)
Comment (swapnili) [reply lib/muser_priv.h:38]: This is the define which is used for unused variables everywhere internally.

File: samples/client.c (hunk 2 above)
Comment (swapnili) [reply samples/client.c:552]: Will fix it.

File: lib/muser_priv.h (hunk 1 above)
Comment (tmakatos) [reply lib/muser_priv.h:38]: I know, why not use directly `__attribute__((unused))`?

Comment (swapnili) [reply lib/muser_priv.h:38]: This way it looks better and readable to me. Also using __attribute__ all over needs extra indentation.

**Review by tmakatos — APPROVED** [review]

**Review by jlevon — APPROVED** [review]

**Pull request merged**